src/track/serato/markers2.h

```diff
@@ -92,102 +116,103 @@ class SeratoMarkers2BpmlockEntry : public SeratoMarkers2Entry {
 
     quint32 length() const override;
 
-private:
+  private:
     bool m_locked;
 };
 
-inline
-bool operator==(const SeratoMarkers2BpmlockEntry& lhs,
-                const SeratoMarkers2BpmlockEntry& rhs) {
+inline bool operator==(const SeratoMarkers2BpmlockEntry& lhs,
+        const SeratoMarkers2BpmlockEntry& rhs) {
     return (lhs.isLocked() == rhs.isLocked());
 }
 
-inline
-bool operator!=(const SeratoMarkers2BpmlockEntry& lhs,
-                const SeratoMarkers2BpmlockEntry& rhs) {
+inline bool operator!=(const SeratoMarkers2BpmlockEntry& lhs,
+        const SeratoMarkers2BpmlockEntry& rhs) {
     return !(lhs == rhs);
 }
 
-inline
-QDebug operator<<(QDebug dbg, const SeratoMarkers2BpmlockEntry& arg) {
+inline QDebug operator<<(QDebug dbg, const SeratoMarkers2BpmlockEntry& arg) {
     return dbg << "locked =" << arg.isLocked();
 }
 
 class SeratoMarkers2ColorEntry : public SeratoMarkers2Entry {
-public:
-    SeratoMarkers2ColorEntry(QColor color)
-        : m_color(color) {
+  public:
+    SeratoMarkers2ColorEntry(QRgb color)
+            : m_color(color) {
     }
 
     SeratoMarkers2ColorEntry()
-        : m_color(QColor()) {
+            : m_color(kDefaultTrackColor) {
     }
 
-    static SeratoMarkers2EntryPointer parse(const QByteArray &data);
+    static SeratoMarkers2EntryPointer parse(const QByteArray& data);
 
     QString type() const override {
         return "COLOR";
     }
 
-    QByteArray data() const override;
+    SeratoMarkers2Entry::TypeId typeId() const override {
+        return SeratoMarkers2Entry::TypeId::Color;
+    }
 
-    QColor getColor() const {
+    QByteArray dump() const override;
+
+    QRgb getColor() const {
```

Comment:
Why QRgb instead of the lightweight and type-safe mixxx::RgbColor?

Reply:
I changed this in PR #2495. I'll rebase this one on it later.